### ISSUE TITLE
not working with TB 38.x

### ISSUE THREAD

**Issue by tessus** [trlkly/dorando-keyconfig#2]
This add-on doesn't show up, if you search for it in TB nor can you install it.

Or is it an extension for FF only? Also, don't new extensions have to be signed?

**Comment by trlkly** [comment]
I haven't tested it on Thunderbird, and so it may not be enabled to work on it. I'll test and see if I can get it to work.

**Comment by trlkly** [comment]
I have tested with Thunderbird 45, and it appears to work fine. I updated the compatibility info on AMO.

In order to find the addon, you will need to search for something like "keyconfig," look under Available Addons, and then scroll down and click  "Show All ___ addons."

This is because it is a preliminarily reviewed Addon. I don't really want to go through the rigamarole a full review when the addon is mostly someone else's code that I only somewhat understand.

I will also include instructions on how to manually install the addon from GitHub.
